### PR TITLE
channelIDs

### PR DESCRIPTION
**Pull request by roiser**
Hi @oliviermattelaer this is the PR for the channelIds, this goes together with the mg5amcnlo:gpucpp_wrap branch. Functionality wise this should be it now. I still mark it as WIP as not all processes have been re-generated and also the comparison of the generated code fails because of some missing spaces here and there. cc @valassi

### CODE REVIEW

**Comment by roiser** [comment]
Dear @oliviermattelaer many thanks for your help with the pkl file, now all tests are green, I suppose we can review this PR at your convenience. cheers

**Comment by oliviermattelaer** [comment]
Hi,

I have looked at the code and this is good to go.
But I have changed the target to master_june24 (as discussed) and as expected they are now a (non super trivial) conflict that we have to fix before being able to do the merge. (and we will fix such conflict tommorow morning)

Good job,

Olivier

**Review by oliviermattelaer — APPROVED** [review]
Let's go.

Thanks a lot !!!

File: epochX/cudacpp/gg_tt.mad/SubProcesses/runTest.cc [hunk 1]
```diff
@@ -222,9 +230,20 @@ struct CUDATest : public CUDA_CPU_TestBase
     constexpr fptype fixedG = 1.2177157847767195; // fixed G for aS=0.118 (hardcoded for now in check_sa.cc, fcheck_sa.f, runTest.cc)
     for( unsigned int i = 0; i < nevt; ++i ) hstGs[i] = fixedG;
     copyDeviceFromHost( devGs, hstGs ); // BUG FIX #566
+<<<<<<< HEAD
```
Comment (valassi) [inline epochX/cudacpp/gg_tt.mad/SubProcesses/runTest.cc:233]: This PR was merged including merge conflicts... See #901

Comment (roiser) [reply epochX/cudacpp/gg_tt.mad/SubProcesses/runTest.cc:233]: This is related to the re-generation of the code for our standard example processes, IIRC we touched on this during the dev meeting yesterday.

**Pull request merged**